src/Core/Defines.h

```diff
@@ -59,6 +59,7 @@
 #define DBMS_MIN_REVISION_WITH_VERSION_PATCH 54401
 #define DBMS_MIN_REVISION_WITH_SERVER_LOGS 54406
 #define DBMS_MIN_REVISION_WITH_CLIENT_SUPPORT_EMBEDDED_DATA 54415
+#define DBMS_MIN_REVISION_WITH_DISTRIBUTED_DEPTH 54417
```

Comment:
This should be equal to new `DBMS_TCP_PROTOCOL_VERSION` value which you should increment manually (in this file below). Better not call it `revision`, `protocol version` is more correct. `revision` is from the old times when we confused the server build revision and the native protocol version, but now they are separate.

Reply:
Ok.